releasenotes/notes/correct-type-annotations-2ee30779966960b6.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Corrected some typing anntotations for PEP 484 compliance
```

```suggestion
    Corrected some typing annotations for PEP 484 compliance
```

https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/10474/workflows/365f7ed0-f811-4c75-b5e9-6d8a43e91972/jobs/800999?invite=true#step-105-349

---

resolved in 275acdc86921bc28b9925e629f3ec4eb2865fcee